docs/streams/upgrade-guide.html

```diff
@@ -117,6 +117,21 @@ <h3><a id="streams_api_changes_300" href="#streams_api_changes_300">Streams API
     <p>
         We removed the default implementation of <code>RocksDBConfigSetter#close()</code>.
     </p>
+    <p>
+        We dropped the default 24 hours grace period for windowed operations such as Window or Session aggregates, or stream-stream joins.
```

Comment:
Why do we include this in this PR? Seems it should be part of KIP-663 doc updates? \cc @izzyacademy

Reply:
Since we are piggy-backing the fix on KIP-663 now, I want to incorporate the change along with this PR.